minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -100,39 +100,35 @@ public void run() {
         try {
           LOG.info("Starting Alluxio master {}.", mMasterProcess);
           mMasterProcess.start();
+        } catch (InterruptedException e) {
+          // this is expected
         } catch (Exception e) {
           // Log the exception as the RuntimeException will be caught and handled silently by JUnit
           LOG.error("Start master error", e);
           throw new RuntimeException(e + " \n Start Master Error \n" + e.getMessage(), e);
         }
       }
     };
-
     mMasterThread = new Thread(runMaster);
-    mMasterThread.setName("MasterThread-" + System.identityHashCode(this));
```

Comment:
we don't do it for any other threads in the local cluster ... I am happy to instead do it everywhere else if you'd prefer